### PR TITLE
add copa and various updates

### PR DESCRIPTION
**Pull request by sozercan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,18 +30,20 @@ Projects listed on this page are open source that Microsoft maintain or contribu
...
...                     | Uses Kubernetes primitives to associate managed identities for Azure resources and identities in Azure Active Directory (AAD) with pods based on [Workload Identity federation](https://docs.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation) | Azure: incubation <br> API: N/A                                                                                                                                                                        | [GitHub issues](https://github.com/Azure/azure-workload-identity/issues)                                                                                                                                                                                                                                                                                                                       | [How to use](https://azure.github.io/azure-workload-identity/docs/installation.html)                                                                                                                                                                                                                                          |
```
Comment (ritazh) [inline README.md:38]: can you pls add a link for WI on AKS: https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview

File: README.md [hunk 2]
```diff
@@ -30,18 +30,20 @@ Projects listed on this page are open source that Microsoft maintain or contribu
...
+| Container Security                   |                                                                                                                                                                                                                                                                                    |                                                                                                                                                                                                                                                                               |                                                                                                                                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (ritazh) [inline README.md:40]: Does this need to be a separate category? or could it be under `Cloud Native Governance and Security` like all the other security projects?

Comment (ritazh) [reply README.md:40]: IMO I think `Cloud Native` is generic enough so we can reuse the same category.

Comment (sozercan) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (ritazh) [reply README.md:38]: @sozercan did you miss this?

**Review by ritazh — APPROVED** [review]
One nit, otherwise LGTM

File: README.md (hunk 1 above)
Comment (sozercan) [reply README.md]: oh i missed this. thanks, updated!

**Pull request merged**